RequestsPlus/RequestsPlus.py

```diff
@@ -122,7 +124,7 @@ def get(self, apiQueryURI, **kwargs):
         :rtype: requests.Response
         """
 
-        response = self._sendRequest(self.methodName(), apiQueryURI, **kwargs)
+        response = self._sendRequest("get", apiQueryURI, **kwargs)
```

Comment:
The mixin from the util package didn't work anymore.  Since it was only used twice it was quicker to hardcode the strings.  IMHO making the strings a property is a good idea but putting more work into it at the moment is over-engineering.